config/renovate.json5

```diff
@@ -109,6 +109,15 @@
             "versioningTemplate": "{{#if versioning}}{{{versioning}}}{{else}}semver-coerced{{/if}}",
             "extractVersionTemplate": "^v(?<version>.*)$"
         },
+        // Matches images where you need to override the image reference in a Helm Value (i.e. https://github.com/defenseunicorns/uds-capability-rook-ceph/blob/f190c32688e80ad72df6389bcb150b35d779d0f0/values/cluster-values.yaml#L3-L4)
+        {
+            "fileMatch": ["\\.*\\.ya?ml$"],
+            "matchStrings": [
+                // Test: https://regex101.com/r/k5ebjz/1
+                "# renovate: image=(?<depName>.*?)\\s*image:\\s.*:(?<currentValue>[^\\s\"]*)",
```

Comment:
This LGTM. Are there any other cases than just `image:` that someone might want to use this functionality? I think you mentioned that just `tag` is covered somewhere else already so maybe that's all.

Reply:
I don't think there are other places that have a full ref that aren't `image` - I have seen `tag` and `version` but those can use the "after-colon" rule that already exists.